charts/spire/charts/tornjak-ui/README.md.gotmpl

```diff
@@ -0,0 +1,43 @@
+{{ template "chart.header" . }}
+
+<!-- This README.md is generated. Please edit README.md.gotmpl -->
+
+{{ template "chart.deprecationWarning" . }}
+
+{{ template "chart.badgesSection" . }}
+[![Development Phase](https://github.com/spiffe/spiffe/blob/main/.img/maturity/dev.svg)](https://github.com/spiffe/spiffe/blob/main/MATURITY.md#development)
+
+{{ template "chart.description" . }}
+
+{{ template "chart.homepageLine" . }}
+
+## Version support
+
+> **Note**: This Chart is still in development and still subject to change the API (`values.yaml`).
+> Until we reach a `1.0.0` version of the chart we can't guarantee backwards compatibility although
+> we do aim for as much stability as possible.
+
+| Dependency | Supported Versions |
+|:-----------|:-------------------|
+| SPIRE      | `1.5.3+`, `1.6.x`  |
+| Helm       | `3.x`              |
+
+## Prerequisites
+This chart requires access to Tornjak API (Backend) component (`tornjakFrontend.apiServerURL`).
+Obtain the URL for Tornjak APIs. If deployed in the same cluster, locally,
+Tornjak APIs are typically available at `http://localhost:10000`. 
+Review Tornjak documentation for more details.
+
+## Usage
+Since this is just a demo version, to access Tornjak APIs you can use 
+port forwarding. See the chart NOTES output for more details.
+
```

Comment:
```suggestion

This chart requires access to Tornjak API (Backend) component (`tornjakFrontend.apiServerURL`).
Obtain the URL for Tornjak APIs. If deployed in the same cluster, locally,
Tornjak APIs are typically available at `http://localhost:10000`. 
Review Tornjak documentation for more details.

## Usage

Since this is just a demo version, to access Tornjak APIs you can use 
port forwarding. See the chart NOTES output for more details.

```